server/router/driver.ts

```diff
@@ -29,23 +33,33 @@ router.get('/:id/profile', (req, res) => {
 });
 
 // Put a driver in Drivers table
-router.post('/', (req, res) => {
+router.post('/', validateUser('Driver'), (req, res) => {
```

Comment:
Did you mean for this to be Dispatcher? Don't we only want Dispatchers adding and removing users?